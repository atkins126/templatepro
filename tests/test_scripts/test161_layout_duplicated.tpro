{{extends "layout01.html"}}

ignored1

{{block "parentblock1"}}
this is the child block 1
{{if value1}}value1 is "true"{{endif}}
{{endblock}}

ignored2

{{block "parentblock1"}}
this is the child block 2
{{if value1}}value1 is "true"{{endif}}
{{endblock}}

ignored3