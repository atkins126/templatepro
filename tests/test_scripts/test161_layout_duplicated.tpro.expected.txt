{{:value1}}

{{:external}}
{{:external.proptrue}}
{{:external.propfalse}}

{{if external }}external is true{{endif}}
{{if !external }}WRONG{{endif}}

{{if external.proptrue }}external.proptrue is true{{endif}}
{{if !external.proptrue }}WRONG{{endif}}


{{if !external.propfalse }}external.propfalse is false{{endif}}
{{if external.propfalse }}WRONG{{endif}}